statsmodels/stats/multivariate.py

```diff
@@ -59,6 +59,41 @@ def test_mvmean(data, mean_null=0, return_results=True):
     else:
         return statistic, pvalue
 
+def test_mvmean_two_sample(x, y, return_results=True):
```

Comment:
my recent convention uses 2indep as postfix `test_mvmean_2indep`
(used in rates and proportions)